### PR TITLE
PMM-6827 DB cluster monitoring.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Review by askomorokhov — APPROVED** [review]
Looks good to me.

File: service/k8sclient/k8client.go [hunk 1]
```diff
@@ -229,7 +231,8 @@ func (c *K8Client) CreateXtraDBCluster(ctx context.Context, params *XtraDBParams
 			},
 
 			PMM: &pxc.PMMSpec{
-				Enabled: false,
+				Enabled:    false,
```
Comment (BupycHuk) [inline service/k8sclient/k8client.go:234]: as we discussed let's enable it and add ServerUser and Image fields.

**Comment by AlekSi** [comment]
A ton of merge conflicts now

**Review by askomorokhov — APPROVED** [review]
Overall looks good to me.
Please rewind FB.

**Review by percona-csalguero — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**